Godeps/_workspace/src/github.com/spf13/pflag/flag.go

```diff
@@ -386,12 +403,16 @@ func (f *FlagSet) FlagUsages() string {
 			// put quotes on the value
 			format = "--%s=%q: %s\n"
 		}
-		if len(flag.Shorthand) > 0 {
+		if len(flag.Shorthand) > 0 && len(flag.ShorthandDeprecated) == 0 {
 			format = "  -%s, " + format
 		} else {
 			format = "   %s   " + format
 		}
-		fmt.Fprintf(x, format, flag.Shorthand, flag.Name, flag.DefValue, flag.Usage)
+		shorthand := flag.Shorthand
+		if len(flag.ShorthandDeprecated) > 0 {
+			shorthand = ""
+		}
+		fmt.Fprintf(x, format, shorthand, flag.Name, flag.DefValue, flag.Usage)
```

Comment:
Hide the shorthand if it's deprecated. (Since we're hiding it, we don't need to put the deprecation message in the help text.)